File: src/AutomatonTheory/JavaLogic/Transition.java
package AutomatonTheory.JavaLogic;

public class Transition {

    State DestinyState = new State();
    String Symbol = "";

    public Transition(){

    }
    public Transition(State destinyState, String symbol){
        DestinyState = destinyState;
        Symbol = symbol;
    }
}

File: src/AutomatonTheory/Logic/Transition.kt
package AutomatonTheory.Logic


open class Transition() {
    open var DestinyState : State = State()
    open var Symbol : String = ""

    constructor(destinyState:State, symbol : String) : this(){
        DestinyState = destinyState
        Symbol = symbol
    }

    constructor(destinyStateName:String, symbol:String) : this(){

    }
}